lib/routes-deprecated/kaopunews/index.js

```diff
@@ -1,28 +0,0 @@
-const got = require('@/utils/got');
```

Comment:
Please remove this entry as well.
https://github.com/DIYgod/RSSHub/blob/cb1ac8e0b4936b8b24943a5689606c0480c37cf8/lib/router.js#L1272